update/update-version.md

```diff
@@ -1,139 +1,187 @@
 # Updating from v3.x.x to v4.x.x
 
-Starting from version v4.x.x, we have refactored the code for the VSI module, resulting in a significant change when transitioning from version 3.x.x to 4.x.x. This means that the Virtual Server Instances (VSIs) managed by this module will be deleted and recreated during the update process.
+Version v4.x.x changes the VSI module code in ways that result in significant changes when you update from version 3.x.x to 4.x.x. When you update, the Virtual Server Instances (VSIs) that are managed by this module are deleted and re-created.
 
-By employing the suggested method, you can avoid the need for VSI recreation.
+Follow these steps to avoid the need to re-create the VSIs when you update.
 
-- [Schematics](#schematics)
-- [Local Terraform State file](#local-terraform-state-file)
+## Before you begin
 
-## Schematics
+Make sure you have recent versions of these command-line prerequisites.
 
-If you have your cloud infrastructure deployed using Schematics, you can use the  `schematics_update_v3.x.x_to_v4.x.x.sh` script to avoid the recreation of Virtual Server Instances (VSIs).
+- [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cli-getting-started)
+- [IBM Cloud CLI plug-ins](https://cloud.ibm.com/docs/cli?topic=cli-plug-ins):
+    - `is` plug-in (vpc-infrastructure)
+    - `sch` plug-in (schematics)
```

Comment:
this is only required if following the "Schematics process"

Reply:
> For IBM Schematics deployments: `sch` plug-in (schematics)